pkg/sources/s3/s3_integration_test.go

```diff
@@ -215,3 +247,48 @@ func TestSource_Validate(t *testing.T) {
 		})
 	}
 }
+
+func TestSourceChunksResumption(t *testing.T) {
+	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
+	defer cancel()
+
+	src := new(Source)
+	src.Progress = sources.Progress{
+		Message:           "Bucket: trufflesec-ahrav-test-2",
```

Comment:
Would you mind setting these tests up against buckets with more descriptive names? I'm worried that someone's going to not realize that they're load-bearing.